### ISSUE TITLE
License ?

### ISSUE THREAD

**Issue by suprafun** [alextrudeau/Genetic-Programming-For-Multi-Robot-Motion-Planning#1]
Hello, under what license is this project released under ? Thank-you.

**Comment by alextrudeau** [comment]
The research associated with this repository has been submitted for publication but hasn't been published yet. Would you mind telling me what you'd like to do with this code? If you just want to experiment without the goal of commercial use or publication, that is fine. Once this research has been published, feel free to do as you please!

**Comment by suprafun** [comment]
I would actually learn from it, to write my own multi-robot motion planning approach for a commercial indie game actually. So will wait : ) Could you please link your repo. to your publication once your research has been published. I am sure that it will be much appreciated by everyone that comes across your project. Thank-you.

**Comment by alextrudeau** [comment]
Yes I'd be happy to link the paper once it's published. I'll also try to document the code and write a proper read-me if I have time (I'm a Masters student haha)
